wallet/db.c

```diff
@@ -177,6 +177,14 @@ char *dbmigrations[] = {
     "ALTER TABLE payments ADD COLUMN payment_preimage BLOB;",
     /* We need to keep the shared secrets to decode error returns. */
     "ALTER TABLE payments ADD COLUMN path_secrets BLOB;",
+    /* Create time-of-payment of invoice, default already-paid
+     * invoices to current time. */
+    "ALTER TABLE invoices ADD paid_timestamp INTEGER;",
+    "CREATE INDEX invoices_paid_timestamp"
+    "    ON invoices(paid_timestamp);",
+    "UPDATE invoices"
+    "   SET paid_timestamp = strftime('%s', 'now')"
+    " WHERE state = 1;",
```

Comment:
Hmm, this is a hard one.  We could leave it as zero, to clearly flag that it was pre-db-update?

Officially, we don't support db updates yet, but OTOH this is all good practice...

Reply:
hmm indeed....